source/PyMaterialX/PyMaterialXGenShader/PyUtil.cpp

```diff
@@ -11,15 +11,21 @@
 namespace py = pybind11;
 namespace mx = MaterialX;
 
+std::vector<mx::TypedElementPtr> findRenderableElements(mx::ConstDocumentPtr doc, bool includeReferencedGraphs)
```

Comment:
Find renderable elements never really worked in Python AFAIK. Fixed it here for the generateShader script.